docs/_docs/tools/control-script.adoc

```diff
@@ -521,6 +521,50 @@ validate_indexes has finished with errors (listed above).
 Like `idle_verify`, index validation tool works correctly only if updates are stopped. Otherwise, there may be a race between the checker thread and the thread that updates the entry/index, which can result in a false positive error report.
 ====
 
+=== Check SQL Index Inline Size
+
+During the operation of the Ignite cluster it could have a different SQL index inline size on cluster nodes.
+For example, due to the `IGNITE_MAX_INDEX_PAYLOAD_SIZE` property value is different on cluster nodes. The difference
+of index inline size may lead to performance drop.
```

Comment:
```suggestion
A running Ignite cluster could have different SQL index inline sizes on its cluster nodes.
For example, it could happen due to the `IGNITE_MAX_INDEX_PAYLOAD_SIZE` property value is different on the cluster nodes. The difference
between index inline sizes may lead to a performance drop.
```

Reply:
I suggest keeping the wording as simple as possible (for example, _During the operation of the Ignite cluster_ = _A running Ignite cluster_ , etc.)